less/mosaico/footer.less

```diff
@@ -0,0 +1,126 @@
+@import (reference) "../ksf-colors";
+
+.mosaico--footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.mosaico-footer {
+  &__links {
+    display: flex;
+    margin: 35px auto 30px;
+    justify-content: center;
+
+    a {
+      margin: 5px 20px;
+      text-decoration: none;
+      color: @dark-text;
+      @media (max-width: @breakpoint-2col) {
+	margin: 5px auto 5px auto;
+      }
+      font: 400 14px Roboto, sans-serif;
+    }
+
+    @media (max-width: @breakpoint-2col) {
+      flex-direction: column;
+    }
+  }
+
+  &__separator {
+    width: 420px;
+    margin: 0 auto 20px;
+    background-color: @grey-slightlylight;
+    border: 0;
+    height: 1px;
+
+    @media (max-width: @breakpoint-2col) {
+      width: 220px;
+    }
+  }
+
+  &__grid {
+    display: grid;
+    grid-template-rows: auto auto;
+    grid-template-columns: 1fr 1fr 1fr;
+    row-gap: 8px;
+    column-gap: 0;
+  }
+
+  &__caption {
+    font: 900 13px "Roboto", sans-serif;
+    color: @grey-mediumlight;
+
+    margin-top: 25px;
+    margin-bottom: 15px;
+  }
+
+  &__other-papers {
+    width: 400px;
+    display: flex;
+
+    @media (max-width: @breakpoint-2col) {
+      width: 250px;
```

Comment:
At least for the mobile view margin bottom or something like that is needed, otherwise the logos don't show entirely